assets/style/services/services.scss

```
@@ -5,3 +5,329 @@
+  min-height: 420px;
+  background-image: url(../../images/services/bg-machines.png);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  @media (max-width: 480px) {
+    min-height: 0;
+  }
+  .main__services-title {
+    font-weight: 700;
+    font-size: 58px;
+    text-align: center;
+    color: $text-light-color;
+    padding: 107px 0 31px 0;
+    @media (max-width: 1024px) {
+      padding: 77px 0 51px 0;
+    }
+    @media (max-width: 864px) {
+      font-size: 45px;
+    }
+    @media (max-width: 480px) {
+      font-size: 35px;
+      padding-top: 30px;
+      padding-bottom: 15px;
+    }
+    @media (max-width: 360px) {
+      font-size: 25px;
+    }
+  }
+  .main__services-title-text {
+    margin-inline: auto;
+    @include default-p($text-light-color);
+    max-width: 892px;
+    text-align: center;
+    @media (max-width: 1400px) {
+      font-size: 30px;
+      line-height: 45px;
+      max-width: none;
+    }
+    @media (max-width: 1024px) {
+      line-height: 35px;
+    }
+    @media (max-width: 864px) {
+      line-height: 30px;
+      font-size: 25px;
+    }
+    @media (max-width: 480px) {
+      font-size: 20px;
+      padding-bottom: 30px;
+    }
+    @media (max-width: 360px) {
+      font-size: 15px;
+      line-height: 18px;
+    }
+  }
+}
+.default-wrapper {
+  @extend %default-wrapper;
+  display: flex;
+  justify-content: center;
+  @media (max-width: 1400px) {
+    flex-wrap: wrap;
+  }
+}
+.main__drilling-work {
+  display: flex;
+  justify-content: space-between;
+  flex: 0 1 50%;
+  @media (max-width: 1400px) {
+    flex-wrap: wrap;
+    flex: 0 1 100%;
+    justify-content: center;
+  }
+}
+.main__drilling-work2 {
+  display: flex;
+  justify-content: space-between;
+  justify-content: center;
+  background-image: url(../../images/services/crystal-background.png);
+  img {
+    padding: 80px 0px;
+  }
+}
+.main__drilling-work-foto {
+  img {
+    padding: 80px 60px 80px 40px;
+    @media (max-width: 1400px) {
+      padding: 40px;
+      padding-top: 60px;
+    }
+    @media (max-width: 768px) {
+      max-width: 100%;
+    }
+    @media (max-width: 480px) {
+      padding: 10px;
+    }
+  }
+}
+.main__drilling-work-foto2 {
+  @media (max-width: 1400px) {
+    order: -1;
+  }
+  img {
+    padding: 80px 0px;
+    @media (max-width: 1400px) {
+      padding: 40px;
+    }
+    @media (max-width: 768px) {
+      max-width: 100%;
+    }
+    @media (max-width: 480px) {
+      padding: 10px;
+    }
+  }
+}
+.main__drilling-work-inform-text {
+  width: 790px;
+  padding: 80px 0px 0px 0px;
+  @media (max-width: 1400px) {
+    text-align: center;
+    padding: 0;
+    width: 100%;
+  }
+}
+.main__drilling-work-inform-text2 {
+  padding: 80px 60px 0px 0px;
+  width: 790px;
+  @media (max-width: 1400px) {
+    padding: 0;
+    width: 100%;
+    text-align: center;
+  }
+}
+.main__drilling-work-inform-text-h2 {
+  @include default-h($text-dark-color);
+  padding-bottom: 15px;
```

Вообще, здесь нет разницы, но лучше задавать в подобных ситуациях марджины, потому что если у тебя, например, был бы задан какой-то цвет на заднем фоне, то у тебя бы этот цвет был виден на твоём паддинге.
```suggestion
  margin-bottom: 15px;
```